lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -838,11 +865,17 @@ sub fix_tests{
     my $table = $check->table;
     my $where = $check->id_where;
     my @edits = @{$check->columns};
+    # If we are inserting and id is displayed, we want to insert
+    # at this exact location
```

Comment:
I can see that  the line below asserts whether we're inserting and the id column is being displayed, but what's unclear to me is what `this exact location` means. Isn't the id displayed so we insert with that exact id-value?